src/styles/abstracts/_functions.scss

```
@@ -2,6 +2,15 @@
   @return $value / ($value * 0 + 1);
 }
 
-@function rem($value) {
-  @return #{strip-unit($value) / strip-unit(16px)}rem;
-}
+@function rem($values) {
+  $max: length($values);
+  $remValues: '';
+  @for $i from 1 through $max {
+    $value: (nth($values, $i));
+    $remValues: #{strip-unit($value) / strip-unit(16px)}rem;
```

Need to add `$value` for `remValues` in order to work properly.
```suggestion
    $value: strip-unit(nth($values, $i));
    $remValues: #{$remValues + $value}rem;
```